common/model-views.h

```diff
@@ -725,4 +791,106 @@ namespace rs2
         std::queue<event_information> _changes;
         std::mutex _mtx;
     };
+
+    enum special_folder
+    {
+        user_desktop,
+        user_documents,
+        user_pictures,
+        user_videos,
+        temp_folder
+    };
+
+    inline std::string get_timestamped_file_name()
+    {
+        std::time_t now = std::time(NULL);
+        std::tm * ptm = std::localtime(&now);
+        char buffer[16];
+        // Format: 20170529_205500
+        std::strftime(buffer, 16, "%Y%m%d_%H%M%S", ptm);
+        return buffer;
+    }
+    inline std::string get_folder_path(special_folder f)
```

Comment:
Why is it inline?

Reply:
It's a free function in a header file...I think it is preferred.